dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -296,12 +299,20 @@ else if ((cmd != null) && cmd.equals(com.dotmarketing.util.Constants.FULL_DELETE
 				Logger.debug(this,"Calling Full Delete Method");
 				WebAsset webAsset = (WebAsset) req.getAttribute(WebKeys.TEMPLATE_EDIT);
 
-				StringBuilder error = new StringBuilder();
+				SessionDialogMessage error = new SessionDialogMessage(
+						LanguageUtil.get(user, "Delete-Template"),
+						LanguageUtil.get(user, "message.template.full_delete.error"),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "message.template.full_delete.error" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)